backend/dataall/core/resource_threshold/db/resource_threshold.py

```diff
@@ -0,0 +1,12 @@
+from dataall.base.db import Base, utils
+from sqlalchemy import String, Integer, Column, Date
+from datetime import date
+
+
+class ResourceThreshold(Base):
+    __tablename__ = 'resource_threshold'
+    actionId = Column(String(64), primary_key=True, default=utils.uuid('resource_threshold'))
+    username = Column(String(64), nullable=False)
+    actionType = Column(String(64), nullable=False)
+    date = Column(Date, default=date.today, nullable=False)
```

Comment:
very nit: `date` is a field that is to understand at first sight. What date is it? We can add a comment or rename it so anyone understand what this column represents

Reply:
I'll change `actionId` to `actionUri` 

For `date` - what better name would you suggest?